airflow/www/templates/appbuilder/flash.html

```diff
@@ -0,0 +1,68 @@
+<!-- Adapted from: https://github.com/dpgaspar/Flask-AppBuilder/blob/master/flask_appbuilder/templates/appbuilder/flash.html -->
+
+<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/flash.css') }}">
+
+<!-- Split array into two arrays: one about Broken DAG, another normal alert  -->
+{% with messages = get_flashed_messages(with_categories=true) %}
+    {% if messages %}
+
+        {% set broken_dag_messages = [] %}
+        {% set other_messages = [] %}
+
+        {% for category, m in messages %}
+            {% if m.startswith('Broken DAG') %}
```

Comment:
What is `category` here? Can we set that when we put the flash so that we don't have to depend on string-prefix matching if we change the format of that "broken dag" message

Reply:
Oh it's warning/error isn't it?

Reply:
Hi @ashb thanks for reviewing. Instead of matching string-prefix we could create another `category` like `quiet_error` or `collapsible_error`, and flash the Broken DAG error in airflow/www/views.py with that category instead. (Others could also flash messages with that category elsewhere.) Then in the flash.html file, we could filter for that new category and apply the alert-error css so those messages have the red background in the collapsible. What do you think of this approach?

We’ll also add in the missing Apache license headers that led to the build fail.

Reply:
I'd say lets be more specific and call categorise them as `dag_import_error`. (It would be nice to _also_ show the stacktrace of the the import errors, but to make that not be spammy visually we need to collapse the stacktrace for each import error too.)

Reply:
Great, I’ve added `dag_import_error` as the new category name in `views.py` and updated the rendering code in `flash.html`. We agree that having the stack trace would be beneficial. If it’s OK, we’d like to do that in a follow-up pull request, so we can get this unit of work in first.